server/controllers/list.js

```diff
@@ -309,7 +316,7 @@ const updateListItem = (req, resp) => {
     {
       _id: listId,
       'items._id': itemId,
-      $or: [{ ownerIds: req.user._id }, { memberIds: req.user._id }]
+      $or: [{ ownerIds: userId }, { memberIds: userId }]
```

Comment:
`memberIds: userId` will be enough, each owner is already a member.

Reply:
this the case, where we been talking about on monday.
Without this $or, toggling doesnt work. I think I can fix in comming PR, where I'm changing all callbacks to promises.